### ISSUE TITLE
`jupyterlab-git` incorrectly responds to filebowser events of other extensions

### ISSUE THREAD

**Issue by jhgoebbert** [jupyterlab/jupyterlab-git#1014]
When using a JupyterLab extension like `jupyterlab-gitlab` (https://gitlab.com/beenje/jupyterlab-gitlab/-/tree/jupyterlab-3)   
which inherits from the JupyterLab file browser class,  
then `jupyterlab-git` seems to be mistakenly notified of that file browser events.

Output from webbrowser's console:
```
Fail to find Git top level for path GitLab:jupyter4jsc/j4j_notebooks.
Error: Traceback (most recent call last):
  File "/lib/python3.8/site-packages/jupyterlab_git/git.py", line 144, in execute
    code, output, error = await current_loop.run_in_executor(
  File "/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/lib/python3.8/site-packages/jupyterlab_git/git.py", line 110, in call_subprocess
    process = subprocess.Popen(
  File "/lib/python3.8/subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/lib/python3.8/subprocess.py", line 1702, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: '/home/goebbert1/gitlab:/jupyter4jsc/j4j_notebooks'
```

**Comment by fcollonval** [comment]
Thanks for reporting the bug @jhgoebbert 

Could you provide the version of the extension you are using? There has been some recent changes regarding tracking the filebrowser in #1004 and #1009 that may trigger or solve this depending on the version you are using.

**Comment by jhgoebbert** [comment]
Hi @fcollonval,  
thank you for your reply.  

Our current setup includes:
- JupyterLab 3.1.6
- jupyterlab-git 0.32.1
- jupyterlab-gitlab, branch jupyterlab-3 (very close to jupyterlab-github)

**Comment by fcollonval** [comment]
I'm looking into it. And I can confirm from the implementation of those extensions that #1004 and #1009 are to blame. I'm reverting them as soon as I can confirm (test) it works with `jupyterlab-gitlab`.

**Comment by fcollonval** [comment]
@jhgoebbert a release a patched version (0.32.2). Would you mind checking it works for you?
